bin/install.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eu
+
+# Ensure PHP is installed
+if ! command -v php > /dev/null 2>&1; then
+  echo "PHP is not found, installation aborted!"
+  exit 1
+fi
+
+# Ensure Composer is installed/updated
+if ! command -v composer > /dev/null 2>&1; then
+  echo "*** Installing Composer ***"
+  php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```

Comment:
This is taken directly from the official Composer documentation. I do agree, nevertheless, this is how it is documented by Composer (which has changed a lot in the last couple of years).